pkgs/by-name/kr/kraken2/package.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitHub, rsync, perl, wget, gnused, findutils, libgcc, llvmPackages, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "kraken2";
+  version = "2.1.4";
+
+  src = fetchFromGitHub {
+    owner = "DerrickWood";
+    repo = pname;
```

```suggestion
    repo = "kraken";
```
Do not rely on repo and program names match, it can change.